src/yang/ietf-babel.yang

```diff
@@ -545,48 +560,47 @@ module ietf-babel {
             "RFC ZZZZ: Babel Information Model, Section 3.3.";
         }
 
-	leaf-list dtls-certs {
-	  type leafref {
-	    path "../../dtls/name";
-	  }
-	  description
-	    "List of references to the babel-dtls-cert-sets entries
-             that apply to this interface.  When an interface instance
-             is created, all babel-dtls instances with
-             babel-dtls-default-apply 'true' will be included in
-             this list.";
-	  reference
-	    "RFC ZZZZ: Babel Information Model, Section 3.3.";
-	}
+        leaf-list dtls-certs {
+          type leafref {
+            path "../../dtls/name";
+          }
+          description
+            "List of references to the dtls entries that apply to
+             this interface.  When an interface instance
+             is created, all dtls instances with default-apply
+             'true' will be included in this list.";
+          reference
+            "RFC ZZZZ: Babel Information Model, Section 3.3.";
+        }
 
-	leaf dtls-cached-info {
-	  type boolean;
-	  description
-	    "Indicates whether the cached_info extension is included
+        leaf dtls-cached-info {
+          type boolean;
+          description
+            "Indicates whether the cached_info extension is included
              in ClientHello and ServerHello packets. The extension
              is included if the value is 'true'.";
-	  reference
-	    "RFC ZZZZ: Babel Information Model, Section 3.3.";
+          reference
+            "RFC ZZZZ: Babel Information Model, Section 3.3.";
 	}
 
-	leaf-list dtls-cert-prefer {
-	  type leafref {
-	    path "../../dtls/certs/type";
-	  }
-	  ordered-by user;
-	  description
-	    "List of supported certificate types, in order of
+        leaf-list dtls-cert-prefer {
+          type leafref {
+            path "../../dtls/certs/type";
+          }
+          ordered-by user;
+          description
+            "List of supported certificate types, in order of
              preference. The values MUST be among those listed in the
-             babel-dtls-cert-types parameter. This list is used to
-             populate the server_certificate_type extension in a
-             Client Hello.  Values that are present in at least one
-             instance in the babel-dtls-certs object of a referenced
-             babel-dtls instance and that have a non-empty
-             babel-cert-private-key will be used to populate the
+             types parameter in the list of certs under dtls. This
```

Comment:
No, babel-dtls-cert-types (old) should simply become dtls-cert-types, which is a parameter in the root object:
       leaf-list dtls-cert-types {
         type identityref {
           base dtls-cert-types;

s/types parameter in the list of certs under dtls/dtls-cert-types parameter/

Reply:
s/types parameter in the list of certs under dtls/dtls-cert-types/